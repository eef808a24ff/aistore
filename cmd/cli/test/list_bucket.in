ais gen-shards --bucket $BUCKET_1 --ext .tar --template="tmp/shard-{0..4}" --fcount 1 --fsize 1KB --conc 1 --cleanup // IGNORE
ais ls $BUCKET_1
ais ls $BUCKET_1/
ais ls $BUCKET_1
ais ls $BUCKET_1 --props=name,size,version
ais ls $BUCKET_1 --props=name,name,size,size,version
ais ls $BUCKET_1 --props=all
ais ls ais://$BUCKET_1/
ais ls cloud://$BUCKET_1/ // FAIL "bucket with name "$BUCKET_1" does not exists"
ais ls gs:// // IGNORE
ais ls s3:// // IGNORE
ais ls aiss://$BUCKET_1 // FAIL "Invalid bucket provider "aiss"."
ais rm bucket $BUCKET_1
