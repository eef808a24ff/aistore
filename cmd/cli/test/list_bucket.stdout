NAME			 SIZE
tmp/shard-0.tar		 2.50KiB
tmp/shard-1.tar		 2.50KiB
tmp/shard-2.tar		 2.50KiB
tmp/shard-3.tar		 2.50KiB
tmp/shard-4.tar		 2.50KiB
NAME			 SIZE
tmp/shard-0.tar		 2.50KiB
tmp/shard-1.tar		 2.50KiB
tmp/shard-2.tar		 2.50KiB
tmp/shard-3.tar		 2.50KiB
tmp/shard-4.tar		 2.50KiB
NAME			 SIZE
tmp/shard-0.tar		 2.50KiB
tmp/shard-1.tar		 2.50KiB
tmp/shard-2.tar		 2.50KiB
tmp/shard-3.tar		 2.50KiB
tmp/shard-4.tar		 2.50KiB
NAME			 SIZE	 VERSION
tmp/shard-0.tar		 2.50KiB	 1
tmp/shard-1.tar		 2.50KiB	 1
tmp/shard-2.tar		 2.50KiB	 1
tmp/shard-3.tar		 2.50KiB	 1
tmp/shard-4.tar		 2.50KiB	 1
NAME			 SIZE	 VERSION
tmp/shard-0.tar		 2.50KiB	 1
tmp/shard-1.tar		 2.50KiB	 1
tmp/shard-2.tar		 2.50KiB	 1
tmp/shard-3.tar		 2.50KiB	 1
tmp/shard-4.tar		 2.50KiB	 1
NAME		 SIZE			 CHECKSUM			 ATIME	 VERSION	 CACHED	 TARGET URL			 STATUS	 COPIES
^tmp/shard-0.tar	 2.50KiB	 (.*)	 .{16}	 1	 yes		 https?://(.*)	 ok	 1$
^tmp/shard-1.tar	 2.50KiB	 (.*)	 .{16}	 1	 yes		 https?://(.*)	 ok	 1$
^tmp/shard-2.tar	 2.50KiB	 (.*)	 .{16}	 1	 yes		 https?://(.*)	 ok	 1$
^tmp/shard-3.tar	 2.50KiB	 (.*)	 .{16}	 1	 yes		 https?://(.*)	 ok	 1$
^tmp/shard-4.tar	 2.50KiB	 (.*)	 .{16}	 1	 yes		 https?://(.*)	 ok	 1$
NAME			 SIZE
tmp/shard-0.tar		 2.50KiB
tmp/shard-1.tar		 2.50KiB
tmp/shard-2.tar		 2.50KiB
tmp/shard-3.tar		 2.50KiB
tmp/shard-4.tar		 2.50KiB
"$BUCKET_1" bucket destroyed
